src/routes/safe/components/Apps/hooks/useSafeAppFromManifest.ts

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react'
+import { logError, Errors } from 'src/logic/exceptions/CodedException'
+import { FETCH_STATUS } from 'src/utils/requests'
+import { getAppInfoFromUrl, getEmptySafeApp } from '../utils'
+import { SafeApp } from '../types'
+
+type UseSafeAppFromManifestReturnType = {
+  safeApp: SafeApp
+  isLoading: boolean
+}
+
+const useSafeAppFromManifest = (appUrl: string): UseSafeAppFromManifestReturnType => {
+  const [safeApp, setSafeApp] = useState<SafeApp>(() => getEmptySafeApp(appUrl))
+  const [status, setStatus] = useState<FETCH_STATUS>(FETCH_STATUS.NOT_ASKED)
```

Comment:
IMO it's better to stick to one thing, either status or isLoading. There's no need for status if we only export `isLoading`

Reply:
done